src/test/java/spoon/test/imports/testclasses/badimportissue3320/source/TestSource.java

```diff
@@ -0,0 +1,37 @@
+package spoon.test.imports.testclasses.badimportissue3320.source;
+
+import spoon.test.imports.testclasses.badimportissue3320.source.other.SomeObjectDto;
+import spoon.test.imports.testclasses.badimportissue3320.source.other.TestAnnotation;
+
+/**
+ * @author Gibah Joseph
+ * Email: gibahjoe@gmail.com
+ * Apr, 2020
+ **/
```

Comment:
Personally i hate reading this. It feels like useless information, which get outdated easily. Git can provide this information better. Maybe remove it?